.circleci/config.yml

```diff
@@ -313,6 +376,44 @@ jobs:
     steps:
       - clair-scan:
           image_name: "astronomerinc/ap-airflow:<< parameters.airflow_version >>-<< parameters.distribution_name >>"
+  scan-trivy:
+    docker:
+      - image: docker:18.09-git
+    description: "Trivy: Vulnerability scanner a Docker image"
+    parameters:
+      airflow_version:
+        description: "The Airflow version, for example '1.10.5'"
+        type: string
+      distribution_name:
+        description: "The base distribution of the container"
+        type: string
+      distribution:
+        description: "The distribution without onbuild"
+        type: string
+    steps:
+      - checkout
+      - setup_remote_docker
+      - attach_workspace:
+          at: /tmp/workspace
+      - run:
+          name: Load archived Docker image
+          command: docker load -i /tmp/workspace/saved-images/astronomerinc/ap-airflow:<< parameters.airflow_version >>-<< parameters.distribution_name >>.tar
+      - run:
+          name: Install trivy
+          command: |
+            apk add --update-cache --upgrade curl rpm
+            curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/master/contrib/install.sh | sh -s -- -b /usr/local/bin
+      - restore_cache:
+          keys:
+            - trivy-cache
+      - run:
+          name: Scan the local image with trivy
+          command: |
+            trivy --ignorefile ./<< parameters.airflow_version >>/<< parameters.distribution >>/.trivyignore --cache-dir /tmp/workspace/trivy-cache --ignore-unfixed -s HIGH,CRITICAL --exit-code 1 --no-progress "astronomerinc/ap-airflow:<< parameters.airflow_version >>-<< parameters.distribution_name >>"
```

Comment:
Since we are specifying it, let's make the file `trivyignore` instead of `.trivyignore`

Reply:
done.